website/docs/dev_api_python.md

```diff
@@ -0,0 +1,45 @@
+---
+id: dev_api_python
+title: AYON Python API
+sidebar_label: Python API
+description: Information about AYON Python API
+toc_max_heading_level: 5
+---
+
+## About AYON Python API
+
+A Python client for connecting to AYON server. The client is using REST and GraphQl to communicate with server with `requests` module.
```

Comment:
Not sure if matters, but graphql is in fact working via REST api too. It is REST endpoint.

Reply:
As far as I know, the implementation details of graphQL, the way it works and its resolvers is different from the fact that it provides a different communication channel with AYON Server. 
and `ayon-python-api` is using both REST API and GraphQl calls.

what do you think?

Reply:
A Python client for connecting to AYON server using `requests` module. Wraps REST API endpoints including GraphQl to communicate with server.

Reply:
https://github.com/ynput/ayon-documentation/pull/239/commits/90f6267567c3e36d417f0d337df9bc43f345b934